_playwright-tests/UI/TemplateCRUD.spec.ts

```diff
@@ -0,0 +1,98 @@
+      await page.getByRole('button', { name: 'Select architecture' }).click();
+      await page.getByRole('option', { name: 'aarch64' }).click();
+      await page.getByRole('button', { name: 'Select version' }).click();
+      await page.getByRole('option', { name: 'el9' }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      const modalPage = page.getByTestId('add_template_modal');
+      const rowRHELRepo = await getRowByNameOrUrl(modalPage, smallRHRepo);
+      await rowRHELRepo.getByLabel('Select row 0', { exact: true }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await modalPage.getByRole('textbox', { name: 'Filter by name/url' }).fill(repoName);
+      const rowRepo = await getRowByNameOrUrl(modalPage, repoName);
+      await rowRepo.getByLabel('Select row 0', { exact: true }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByText('Use latest content', { exact: true }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByText('add template modal', { exact: true });
+      await page.getByPlaceholder('Enter name').fill(`${templateName}`);
+      await page.getByPlaceholder('Description').fill('Template test');
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByRole('button', { name: 'Create other options' }).click();
+      await page.getByText('Create template only', { exact: true }).click();
+    });
+    await test.step('Read and update values in the template', async () => {
+      const rowTemplate = await getRowByNameOrUrl(page, templateName);
+      await rowTemplate.getByRole('button', { name: templateName }).click();
+      await expect(page.getByLabel('Breadcrumb').first()).toHaveText('RHELContentTemplates');
+      await expect(page.getByRole('heading', { level: 1 })).toHaveText(templateName);
+      await expect(page.getByText('Description:Template test')).toBeVisible();
+      await page.getByRole('button', { name: 'Actions' }).click();
+      await page.getByRole('menuitem', { name: 'Edit' }).click();
+      await expect(page.getByText('Define template content')).toBeVisible();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(
+        page.getByText('Additional Red Hat repositories', { exact: true }),
+      ).toBeVisible();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Custom repositories', { exact: true })).toBeVisible();
+      await expect(page.getByText(`${repoName}`)).toBeVisible();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      const modalTitle = page.getByTestId('set_up_date');
+      await expect(modalTitle.getByText('Set up date', { exact: true })).toBeVisible();
```

Comment:
```
Error: expect.toBeVisible: Error: strict mode violation: getByText('Set up date', { exact: true }) resolved to 2 elements:
        1) <button aria-current="step" id="set_up_date_step" data-ouia-safe="true" data-ouia-component-type="PF5/WizardNavItem" class="pf-v5-c-wizard__nav-link pf-m-current" data-ouia-component-id="OUIA-Generated-WizardNavItem-5">Set up date</button> aka getByRole('button', { name: 'Set up date' })
        2) <h1 data-ouia-safe="true" class="pf-v5-c-title pf-m-2xl" data-ouia-component-type="PF5/Title" data-ouia-component-id="set_up_date">Set up date</h1> aka getByRole('heading', { name: 'Set up date' })
```

Reply:
That is very strange, it passes for me. 
Also here: https://app.currents.dev/instance/8TSLsfSMOrTQhZDV

I'll test with `getByRole('heading', { name: 'Set up date' })`

Reply:
`getByRole('heading'` works, and I have changed to "getByRole('heading'," in a few places for consistency.

Reply:
Yep that's it ! These changes works !